src/components/UserPortal/OrganizationCard/OrganizationCard.tsx

```
@@ -11,14 +11,12 @@ import {
 } from 'GraphQl/Mutations/OrganizationMutations';
 import { useMutation, useQuery } from '@apollo/client';
 import {
-  USER_JOINED_ORGANIZATIONS,
-  USER_ORGANIZATION_CONNECTION,
-} from 'GraphQl/Queries/OrganizationQueries';
-import useLocalStorage from 'utils/useLocalstorage';
+  ORGANIZATION_LIST,
+  USER_JOINED_ORGANIZATIONS_PG,
+} from 'GraphQl/Queries/Queries';
+import useLocalStorage, { getItem } from 'utils/useLocalstorage';
```

ESLint

[error] 'useLocalStorage' is defined but never used.

(@typescript-eslint/no-unused-vars)